### PR TITLE
feat(unsplash): allow user to select multiple topics

### PR DESCRIPTION
**Pull request by nbouliol**
## Subtitle of PR

Implements #97

The user can select one or more topics

The selected topics are send to unsplash api that accepts multiple topics

From https://unsplash.com/documentation#get-a-random-photo
> topics | Public topic ID(‘s) to filter selection. If multiple, comma-separated

### Developer Testing Evidence

Evidence of the PR is in action in the section below.

- [ ] Tested in Chrome
- [X] Tested in Firefox
- [ ] Tested in Edge

### Firefox
image

### CODE REVIEW

**Comment by nbouliol** [comment]
It will also enable users to scroll horizontally in the settings inputs (because it was required for the select multiple)

For example in the CSS Snippet textarea

File: yarn.lock
Comment (the-wright-jamie) [inline yarn.lock:1]: You left your `yarn` lock file in, please remove 🙃

**Review by the-wright-jamie — CHANGES_REQUESTED** [review]
Looking good, will dev test this myself just to be safe and then merge. In the meantime, please remove the `yarn` lock file 😛

**Comment by the-wright-jamie** [comment]
Your change has just been published to stores and you should see it locally soon.

For your notes, I made a small modification to your note about selecting multiple topics as I couldn't get it to work without holding down Control. You'll see it, but it looks like this:

| Windows/Linux et al | macOS |
| :-: | :-: |
| image | image |

Thank you once again for your contribution!

**Pull request merged**